extensions/amp-form/0.1/amp-form.css

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software'
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-form .amp-hidden {
```

Comment:
Currently it's triggered internally when the user submits the form and the success or error message is displayed in-place of the form. You can [try it out](https://amp-mk-1.herokuapp.com/examples.build/forms.amp.max.html) here.